### ISSUE TITLE
Add Build Instructions

### ISSUE THREAD

**Issue by BenMiller3** [PhantomBadger/SuperAutoPetsMod#1]
The progress on this is amazing! Would love to give it a try locally but can't seem to get it running. Awesome work so far!

**Comment by PhantomBadger** [comment]
Yeah there's a fair bit of setup required to get things in a working state at the moment. I'm heading on vacation soon but when I'm back I'll be working on getting this ready for a "v1.0" release, after which I'll get some build steps and some usable .DLLs/install guide available 

I'll be sure to update this issue when it's ready

**Comment by PhantomBadger** [comment]
As an update on this - I'm functionally nearly ready as far as 'code cleanup' is concerned, but SAP's newer update (to a newer unity version) has broken BepInEx - which is what I use to inject the mod and interface with mono. Once this is resolved I'll work towards the v1.0 Release*.

_*I'm actually getting married in like <2 weeks so hey might not be until mid July_

I don't anticipate much long term support for this mod after that, but once it's set up with build/usage steps it'll be easier for people to tinker if anything needs upgrading in future.

**Comment by BenMiller3** [comment]
Awesome, thanks for the update! And congratulations! I've honestly just needed the capabilities to be able to manipulate a few things within the base SAP so the build instructions would be all I need, and will hopefully be helpful to future contributors of this project as well. Cheers mate

**Comment by PhantomBadger** [comment]
The mod I'm currently looking to implement is based off of BepInEx, you should be able to follow the instructions specified here: https://docs.bepinex.dev/master/articles/user_guide/installation/unity_il2cpp.html

As for building this project specifically, after unhollowing the game via the above instructions, the unhollowed .dlls are copied to a local folder per the instructions here https://docs.bepinex.dev/master/articles/dev_guide/plugin_tutorial/2_plugin_start.html (specifically, see the notes under 'Referencing from local install').

All the build .dlls go into a plugins folder in the BepInEx folder for the project.

That *should* be enough to get everything up and running, as mentioned however SAP is now in too new of a unity version for BepInEx, so you wont be able to get further than that. But if you wanted to roll back steam version and get anything working in the meantime whilst I have this on hold thats your best bet
